packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointUtil.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export class CheckpointUtil {
+  // If the valueInput is a string then add quoutes to both inputs
+  // "<nameInput>": "<valueInput>"
+  // If the valueInput is a number or boolean then only add quotes to the name
+  // "<nameInput>": <valueInput>
+  public static formatNameValuePairForJSon(
```

Comment:
This seems more complicated than necessary. See my earlier comment about using `JSON.stringify()` which should take care of the quotes for you.